contracts/FTokenStrat.sol

```diff
@@ -90,6 +107,10 @@ contract FTokenStrat is IStrat {
                 .div(10**fToken.decimals());
     }
 
+    function totalRewardTokenPending() public returns (uint) {
+        return rewardpool.rewards(address(this));
+    }
+
     function calcTotalValue() external view override returns (uint) {
```

Comment:
Seems like staked amount (which should be all of the invested) is not represented here.